yogadl/rw_coordinator/_client.py

```diff
@@ -44,9 +48,10 @@ def __init__(
             self.ctx.load_verify_locations(cafile=coordinator_cert_file)
         else:
             self.ctx.load_default_certs()
+        self._coordinator_cert_name = coordinator_cert_name
```

Comment:
non-blocking: Not _strictly_ related, but thematically similar and useful: Could you perhaps update the lines above this with the fix from [this other PR](https://github.com/determined-ai/determined/pull/1409/files)?

Reply:
done.